js/foundation.orbit.js

```diff
@@ -239,6 +241,44 @@ class Orbit {
   }
 
   /**
+   * Detaches event listeners
+   */
+  _detachEvents() {
+    // Don't do anything if there are no slides (first run)
+    if (typeof this.$slides == "undefined") {
+      return;
+    }
+
+    if (this.$slides.length > 1) {
+      // Remove old events
+      this.$element.off('.zf.orbit').find('*').off('.zf.orbit')
+
+      // Restart timer if autoPlay is enabled
+      if (this.options.autoPlay) {
+        this.timer.restart();
+      }
+
+      // Reset all sliddes
+      this.$slides.each(function(el) {
+        $(el).removeClass('is-active is-active is-in')
+          .removeAttr('aria-live')
+          .hide();
+      });
+
+      // Show the first slide
+      this.$slides.first().addClass('is-active').show();
+
+      // Triggers when the slide has finished animating
+      this.$element.trigger('slidechange.zf.orbit', [this.$slides.first()]);
```

Comment:
I am not sure this is necessary, but users might be listening to this event and by switching to the first slide, this must be triggered.